dev-scripts/lagoon-get-backup.sh

```diff
@@ -3,24 +3,28 @@
 set -eo pipefail
 
 LAGOON_ENVIRONMENT="${LAGOON_ENVIRONMENT:-main}"
+# The entry in the list of backups for the provided type to retrieve for the
+# given backup type. 1 is the most recent.
+BACKUP_ENTRY_INDEX=$((${BACKUP_ENTRY:-1}-1));
 
 if [[ -z "${LAGOON_PROJECT}" || -z "${BACKUP_TYPE}" || -z "${BACKUP_DESTINATION}" ]]; then
 	echo "usage: LAGOON_PROJECT=<LAGOON_PROJECT> BACKUP_TYPE=<BACKUP_TYPE> BACKUP_DESTINATION=<BACKUP_DESTINATION> $0" >&2
 	exit 1
 fi
 
 BACKUPS=$(lagoon list backups -p "${LAGOON_PROJECT}" -e "${LAGOON_ENVIRONMENT}" --output-json);
-BACKUP_ID=$(echo "$BACKUPS" | jq -r ".data[] | select(.source == \"${BACKUP_TYPE}\") | .backupid" | head -n 1);
-BACKUP_URL="Error";
-echo -e "\nRetrieving ${BACKUP_TYPE} backup from ${LAGOON_ENVIRONMENT} environment of ${LAGOON_PROJECT} project \n\n";
+BACKUP_ID=$(echo "$BACKUPS" | jq -r ".data | map(select(.source == \"${BACKUP_TYPE}\")) | nth(${BACKUP_ENTRY_INDEX}) | .backupid");
```

Comment:
See comment about ids vs indexes on the other PR.

Reply:
Copy/paste of the comment there:

> I have reconsidered your suggestion. As it is now it may be preferable to use the nth entry. This makes it a bit easier for the caller to retrieve the nth backup where files and database belong together. This is a bit more tricky when using ids as you have to retrieve the list of backups first.
>
> As such (and as this being merged) I will leave this as is.